[INFO] [2021-06-15 21:56:45,747] [3074:139995760379712] - task_executor.py[line:313]: report task 2021061521563264143218_dataio_0 0 host 9998 to driver
[INFO] [2021-06-15 21:56:46,165] [3074:139995760379712] - task_executor.py[line:150]: Run 2021061521563264143218 dataio_0 2021061521563264143218_dataio_0 host 9998 task
[INFO] [2021-06-15 21:56:46,166] [3074:139995760379712] - task_executor.py[line:151]: Component parameters on party {'DataIOParam': {'input_format': 'dense', 'delimitor': ',', 'data_type': 'float64', 'exclusive_data_type': None, 'tag_with_value': False, 'tag_value_delimitor': ':', 'missing_fill': False, 'default_value': 0, 'missing_fill_method': None, 'missing_impute': None, 'outlier_replace': False, 'outlier_replace_method': None, 'outlier_impute': None, 'outlier_replace_value': 0, 'with_label': False, 'label_name': 'y', 'label_type': 'int', 'output_format': 'dense', 'need_run': True}, 'dsl_path': './examples/LDPDev/dsl/v2/hetero_secureboost/test_secureboost_train_dsl.json', 'job_parameters': {'job_type': 'predict', 'work_mode': 0, 'backend': 0, 'computing_engine': 'STANDALONE', 'federation_engine': 'STANDALONE', 'storage_engine': 'STANDALONE', 'engines_address': {'computing': {'nodes': 1, 'cores_per_node': 20}, 'federation': {'nodes': 1, 'cores_per_node': 20}, 'storage': {'nodes': 1, 'cores_per_node': 20}}, 'federated_mode': 'SINGLE', 'task_parallelism': 1, 'computing_partitions': 4, 'federated_status_collect_type': 'PULL', 'model_id': 'guest-9999#host-9998#model', 'model_version': '2021061521542761899517', 'eggroll_run': {'eggroll.session.processors.per.node': 4}, 'spark_run': {}, 'rabbitmq_run': {}, 'pulsar_run': {}, 'adaptation_parameters': {'task_memory_per_node': 0, 'if_initiator_baseline': False, 'task_cores_per_node': 4, 'request_task_cores': 4, 'task_nodes': 1}}, 'component_parameters': {'common': {'hetero_secure_boost_0': {'task_type': 'classification', 'objective_param': {'objective': 'cross_entropy'}, 'num_trees': 3, 'validation_freqs': 1, 'encrypt_param': {'method': 'iterativeAffine'}, 'tree_param': {'max_depth': 3}}, 'evaluation_0': {'eval_type': 'binary'}}, 'role': {'guest': {'0': {'dataio_1': {'with_label': True, 'output_format': 'dense'}, 'reader_0': {'table': {'namespace': 'experiment', 'name': 'default_credit_hetero_guest'}}, 'dataio_0': {'with_label': True, 'output_format': 'dense'}, 'reader_1': {'table': {'name': 'breast_hetero_guest', 'namespace': 'experiment'}}}}, 'host': {'0': {'dataio_1': {'with_label': False}, 'reader_0': {'table': {'namespace': 'experiment', 'name': 'default_credit_hetero_host'}}, 'dataio_0': {'with_label': False}, 'reader_1': {'table': {'name': 'breast_hetero_host', 'namespace': 'experiment'}}}}}}, 'initiator': {'party_id': 9999, 'role': 'guest'}, 'dsl_version': 2, 'role': {'guest': [9999], 'host': [9998]}, 'conf_path': './examples/LDPDev/test_predict_conf.json', 'local': {'role': 'host', 'party_id': 9998}, 'CodePath': 'federatedml/util/data_io.py/DataIO', 'module': 'DataIO', 'output_data_name': ['data']}
[INFO] [2021-06-15 21:56:46,166] [3074:139995760379712] - task_executor.py[line:152]: Task input dsl {'model': ['pipeline.dataio_0.model'], 'data': {'data': ['reader_0.data']}}
[ERROR] [2021-06-15 21:56:52,087] [3074:139995760379712] - task_executor.py[line:191]: fit & transform data' header should be same
Traceback (most recent call last):
  File "/fate/python/fate_flow/operation/task_executor.py", line 168, in run_task
    run_object.run(component_parameters_on_party, task_run_args)
  File "/fate/python/federatedml/model_base.py", line 98, in run
    this_data_output = func(*real_param)
  File "/fate/python/federatedml/util/data_io.py", line 895, in transform
    return self.reader.read_data(data_inst, "transform")
  File "/fate/python/federatedml/util/data_io.py", line 150, in read_data
    data_instance = data_overview.header_alignment(data_instance, fit_header)
  File "/fate/python/federatedml/statistic/data_overview.py", line 75, in header_alignment
    raise ValueError("fit & transform data' header should be same")
ValueError: fit & transform data' header should be same
[INFO] [2021-06-15 21:56:52,236] [3074:139995760379712] - task_executor.py[line:313]: report task 2021061521563264143218_dataio_0 0 host 9998 to driver
[INFO] [2021-06-15 21:56:52,319] [3074:139995760379712] - task_executor.py[line:202]: task 2021061521563264143218_dataio_0 host 9998 start time: 2021-06-15 21:56:45
[INFO] [2021-06-15 21:56:52,319] [3074:139995760379712] - task_executor.py[line:204]: task 2021061521563264143218_dataio_0 host 9998 end time: 2021-06-15 21:56:52
[INFO] [2021-06-15 21:56:52,319] [3074:139995760379712] - task_executor.py[line:206]: task 2021061521563264143218_dataio_0 host 9998 takes 6.633s
[INFO] [2021-06-15 21:56:52,320] [3074:139995760379712] - task_executor.py[line:209]: Finish 2021061521563264143218 dataio_0 2021061521563264143218_dataio_0 0 host 9998 task failed
[INFO] [2021-06-15 21:56:52,342] [3074:139995760379712] - task_executor.py[line:313]: report task 2021061521563264143218_dataio_0 0 host 9998 to driver
